### PR TITLE
SOLR-15955: Jetty 10.x upgrade notes

### PR DESCRIPTION
**Pull request by risdenk**
https://issues.apache.org/jira/browse/SOLR-15955

### CODE REVIEW

File: solr/CHANGES.txt [hunk 1]
```diff
@@ -51,6 +51,8 @@ New Features
 
 * SOLR-16596: Learning To Rank - Added support for null feature values in multiple additive trees models (Anna Ruggero via Alessandro Benedetti)
 
+* SOLR-15955: Upgrade to Jetty 10.x (Mark Miller, Kevin Risden)
+
```
Comment (risdenk) [inline solr/CHANGES.txt:55]: Not sure that under `"New Feature"` makes sense, but figured it was good to call out up top.

**Review by tflobbe — APPROVED** [review]

File: solr/CHANGES.txt (hunk 1 above)
Comment (dsmiley) [reply solr/CHANGES.txt:55]: Definitely not "New Feature" for bumping up a dependency unless somehow the user gets some new ability here.

Comment (risdenk) [reply solr/CHANGES.txt:55]: Thats fair - the only existing place that made rough sense was "Other Changes" or "Build" based on the existing categories being:

* New Features
* Improvements
* Optimizations
* Bug Fixes
* Build
* Other Changes

Comment (dsmiley) [reply solr/CHANGES.txt:55]: Typically goes in Other Changes.  Could be justified in some other categories if we expect an appreciable difference.
Obviously this is a big dependency upgrade so it should be called out when we do the release notes, maybe mentioning some benefits if you know of any in particular that matter.

**Review by epugh — APPROVED** [review]
Leaving aside the "New Feature" discussion, the rest makes sense.

File: solr/CHANGES.txt (hunk 1 above)
Comment (risdenk) [reply solr/CHANGES.txt]: moved to other changes

**Pull request merged**